Bottom: 60fdc48f62159d774432e98290340ca148fd11e5
Top:    0d299d4377e967f8503d747a129dfe55b9dedc71
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-27 16:25:03 +0100

Refresh of add-support-for-depth-to-trian

---

diff --git a/src/hid/common/hidgl.h b/src/hid/common/hidgl.h
index a42e0be..9dd76e3 100644
--- a/src/hid/common/hidgl.h
+++ b/src/hid/common/hidgl.h
@@ -81,6 +81,5 @@ int hidgl_stencil_bits (void);
 int hidgl_assign_clear_stencil_bit (void);
 void hidgl_return_stencil_bit (int bit);
 void hidgl_reset_stencil_usage (void);
-void hidgl_set_depth (float depth);
 
 #endif /* __HIDGL_INCLUDED__  */
